sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -35,21 +33,12 @@ export async function createRecorder(context: VitestTestContext): Promise<Record
   return recorder;
 }
 
-function getEndpointAndAPIKeyFromResourceType(resourceType: DeploymentType): {
-  endpoint: string;
-  apiKey: string;
-} {
+function getEndpointFromResourceType(resourceType: DeploymentType): string {
   switch (resourceType) {
     case "embeddings":
-      return {
-        endpoint: assertEnvironmentVariable("AZURE_EMBEDDINGS_ENDPOINT"),
-        apiKey: assertEnvironmentVariable("AZURE_EMBEDDINGS_CLIENT_SECRET"),
-      };
+      return assertEnvironmentVariable("AZURE_EMBEDDINGS_ENDPOINT");
     case "completions":
-      return {
-        endpoint: assertEnvironmentVariable("AZURE_ENDPOINT"),
-        apiKey: assertEnvironmentVariable("AZURE_CLIENT_SECRET"),
-      };
+      return assertEnvironmentVariable("AZURE_AAD_ENDPOINT");
```

Comment:
should `AZURE_AAD_ENDPOINT` be renamed to clarify it is for a completion model deployment?

Reply:
i will update the recordings with the new env var name

Reply:
The recordings don't need to be updated just for this, the env var name doesn't show up in them. Apologies if I misunderstood your comment.